scripts/delete-federation.sh

```diff
@@ -56,16 +57,17 @@ done
 # Remove federation CRDs, namespace, RBAC and deployment resources.
 if [[ "${NAMESPACED}" ]]; then
   ${KCD} -n "${NS}" -f hack/install-namespaced.yaml
-  ${KCD} ns "${NS}"
 else
   ${KCD} -f hack/install-latest.yaml
```

Comment:
Here need some update, we cannot always delete `hack/install-latest.yaml`, if we are not using `latest` but customer build docker image, we should delete `hack/install.yaml`, I will handle this in a separate PR.